Saydalia-Online/appsettings.json

```diff
@@ -7,6 +7,6 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "Default": "Server=.; Database= MedTechSaydaliaOnline; Trusted_Connection= True; TrustServerCertificate= True;MultipleActiveResultSets=true"
+    "Default": "Server=DESKTOP-00JH14J\\MSSQLSERVER08; Database= Saydalia-Online; Trusted_Connection= True; TrustServerCertificate= True;MultipleActiveResultSets=true"
```

Comment:
Please handle  it in appsetting.development.json file since we have change the database name you also need to use the same name 
Kindly consider this requirements